### PR TITLE
Fix status update not getting triggered

### PR DESCRIPTION
**Pull request by aaha97**
Old vsvip cache should be fetched before updating the cache

### CODE REVIEW

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

File: internal/rest/avi_obj_vsvip.go [hunk 1]
```diff
@@ -606,13 +606,13 @@ func (rest *RestOperations) AviVsVipCacheAdd(rest_op *utils.RestOp, vsKey avicac
 		}
 
 		k := avicache.NamespaceName{Namespace: rest_op.Tenant, Name: name}
+		oldVsVipCache, oldVsVipFound := rest.cache.VSVIPCache.AviCacheGet(k)
```
Comment (akshayhavile) [inline internal/rest/avi_obj_vsvip.go:609]: @aaha97: I did not get how status update will have impact by moving this line here?

Comment (aaha97) [reply internal/rest/avi_obj_vsvip.go:609]: we are updating the cache on line 610, so calling old cache on line 615 will fetch updated cache and not old cache

**Comment by akshayhavile** [comment]
skip jira-id-check

**Comment by akshayhavile** [comment]
build ako

**Review by akshayhavile — APPROVED** [review]

**Review by pkoshtavmware — APPROVED** [review]
LGTM

**Pull request merged**